proto/mls/message_contents/message.proto

```diff
@@ -22,7 +24,10 @@ message WelcomeMessage {
 message GroupMessage {
   // Version 1 of the GroupMessage format
   message V1 {
-    bytes mls_message_tls_serialized = 1;
+    uint64 id = 1;
```

Comment:
I noticed this proto is being used in the `SendGroupMessages` request. The caller won't know either the `id` or the `created_ns` value at the time of sending. Is it expected that they set those to 0 values? Or should we have a separate message type for new/unsent messages?

Reply:
Will add another type for the query input